tools/vsimporter/include/PBX/PBXNativeTarget.h

```diff
@@ -54,12 +55,14 @@ class PBXNativeTarget : public PBXTarget {
 
     /* Start of serialized values */
     String m_productReferenceId;
+    String m_nativeBuildConfigurationListId;
     String m_productType;
     StringVec m_buildRuleIds;
     /* End of serialized values */
 
     PBXFileReference* m_productReferencePtr;
     BuildRuleList m_buildRulePtrs;
+    XCConfigurationList* m_nativeBuildConfigurationListPtr;
```

Comment:
We do not, it is just a pointer to an object owned by the pbxDoc.  Ideally it would be a shared_ptr but that's a lot more fixing than I want to do for this change set.